content/en/error_tracking/backend/getting_started.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Backend Error Tracking
+aliases:
+- /error_tracking/standalone_backend/getting_started
+further_reading:
+- link: '/error_tracking/issue_states/'
+  tag: 'Documentation'
+  text: 'Error Tracking Issue States and Workflows'
+- link: '/error_tracking/backend/exception_replay'
+  tag: 'Documentation'
+  text: 'Simplify production debugging with Datadog Exception Replay'
+- link: "/error_tracking/explorer"
+  tag: "Documentation"
+  text: "Learn about the Error Tracking Explorer"
+---
+
+## Overview
+
+[Error Tracking][1] processes errors collected by the Datadog Agent. Whenever an [error span][2] or span containing an error [span event][3] is collected, Error Tracking processes and groups it under an issue, or group of similar errors.
+
+## Getting started
+
+Follow the [in-app setup instructions][4] or choose an instrumentation approach to start collecting backend errors:
+
+### Using Single Step Instrumentation
+
+If you install or update a Datadog Agent with the **Enable APM Instrumentation** and **Error Tracking Standalone** options selected, the Agent is installed and configured to enable Backend Error Tracking. This allows you to automatically instrument your application, without any additional installation or configuration steps.
+
+The following examples show how it works for each deployment type.
+
+{{< tabs >}}
+{{% tab "Linux host or VM" %}}
+
+For an Ubuntu host:
```

Comment:
Should be the same across many, if not all, supported linux distros, not just ubuntu